scripts/genetic_backtester/darwin.js

```diff
@@ -240,6 +241,23 @@ let RangeNeuralActivation = () => {
 };
 
 let strategies = {
+  bollinger: {
+    period_length: RangePeriod(1, 60, 'm'),
+    min_periods: Range(1, 20),
```

Comment:
Any specific reason why you chose min_periods between 1-20? All the other strategies use 1-120, 1-200